_pages/en/strategy-open-first.md

```diff
@@ -0,0 +1,444 @@
+
+It is important to stress that the principles and the action plan relate to existing work processes.
+The use of OSS is commonplace across the GC and in many parts of the organisation (ESDC) there is an established OSS-like culture.
+We simply need to do more and keep getting better at it.
+
+Below are the six principles that govern the strategy:
+
+### Open First
+
+> Where possible, use open standards and OSS first.
+> Work in the open by default
+
+The GC is an enthusiastic user of and contributor to OSS.
+This is reflected in its Digital Standards and Policy on Service and Digital, which encourages the IT community to tap into the growing potential of open standards and OSS as well as join forces with major commercial players and communities, and mobilise co-creation capacity to support departments in pioneering new solutions.
+
+Note:
+
+By default, all work without classified information should be open.
+Security and privacy methods and tools should be made available for all to ensure that this practice can be accomplished with the least barriers as possible.
+
+### Transform
+
+> We harness the working principles of open source; we innovate and co-create, share and reuse, and together build user-centric, data-driven public services.
+
+We adopt a working culture based on the principles of open, sharing and reuse.
+We nudge ESDC and its working methods forward by highlighting the crucial role of open standards and OSS and related best practices for co-creation and collaboration.
+Ideally, this transformation will go beyond IT, so that we apply open methods when working together and sharing information.
+
+Open standards and OSS offer practical tools and methods for teams to work together and co-create solutions, to share their expertise freely in order to obtain better results and have their solutions in actual use sooner.
+In these teams, which sometimes involve several directorates-general, developers look at software code written by their colleagues and try to insert new components.
+They combine parts, borrow tools and recipes, and together build and extend a collection of solutions they can use to run digital government solutions.
+
+Inner source (the creation of working cultures based on open principles and share code within the organization) will be promoted, in order to bring teams closer to working in the open and encourage sharing and reuse within the organisation.
+This is a step towards opening up more projects as open source code and sharing them with the outside world.
+
+Note:
+
+Culture change? Aligns with DevOps, digital standards, etc.
+
+### Share
+
+> All source code must be released under an appropriate OSS license.
+> We enable incidental contributions to related open standards and OSS projects.
+
+Wherever it makes sense to do so, the ESDC will share the source code of its future IT projects.
```

Comment:
ESDC will strive to share the source code by default

Reply:
Yep agree with Guillaume's suggested change for this principle.  Currently the principle text contradicts the principle description 'All source code' vs. 'wherever it makes sense to do so...'